cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/Broker.java

```diff
@@ -202,6 +202,10 @@ public boolean isAlive() {
     return _state != State.DEAD;
   }
 
+  public boolean isUsingJBOD() {
+    return _diskByLogdir.size() > 1;
```

Comment:
1. Can we add a JavaDoc?
2. Nit: `_diskByLogdir.size() > 1` -> `!_diskByLogdir.isEmpty()`

Reply:
If we use `!_diskByLogdir.isEmpty()`, does that mean if `_diskByLogdir` is not empty, the broker must be using JBOD?

I thought `_diskByLogdir.size() == 1` means that the broker is using non-JBOD and `_diskByLogdir .size() > 1`  means that the broker is using JBOD.

Reply:
> If we use `!_diskByLogdir.isEmpty()`, does that mean that the broker is using JBOD if and only if `_diskByLogdir` is NOT empty?

Correct -- please see how we `_diskByLogdir` is initialized in constructor.

Note that if a broker has 1 disk (i.e. non-JBOD), then the cluster model does not add another layer of abstraction -- i.e. keeps track of replicas per broker as opposed to replicas per disk in the broker, which would always be the same disk/logDir in a non-JBOD broker.

Reply:
Got it. Thanks